template.go

```diff
@@ -274,15 +305,16 @@ func (t *Template) Execute(w io.Writer, m map[string]interface{}) (int64, error)
 //
 // This function is optimized for frozen templates.
 // Use ExecuteFuncString for constantly changing templates.
-func (t *Template) ExecuteFuncString(f TagFunc) string {
+func (t *Template) ExecuteFuncString(f TagFunc) (string, error) {
+	var s string
 	bb := t.byteBufferPool.Get()
 	if _, err := t.ExecuteFunc(bb, f); err != nil {
-		panic(fmt.Sprintf("unexpected error: %s", err))
+		return s, err
 	}
-	s := string(bb.Bytes())
+	s = string(bb.Bytes())
```

Comment:
bb typenya apa ya ?
klo buffer bytes mungkin bisa pake `.String()`
```suggestion
	return bb.String(), nil
```

Reply:
dia pake custom struct dia sendiri, jadi gk bisa I guess

Reply:
https://github.com/valyala/bytebufferpool

Reply:
[Ini ada anjing](https://github.com/budiryan/fasttemplate/blob/dcecefd839c4193db0d35b88ec65b4c12d360ab0/vendor/github.com/valyala/bytebufferpool/bytebuffer.go#L105)

Reply:
updated

Reply:
you forget this [one](https://github.com/budiryan/fasttemplate/pull/1/files?utf8=%E2%9C%93&diff=unified#diff-4779dc22f7a7835ae9d75daa70021ad8R97)

Reply:
updated